### PR TITLE
fix ovc & acl tests

### PR DESCRIPTION
**Pull request by ahmedelsayed-93**

### CODE REVIEW

File: functional_testing/Openvcloud/ovc_master_hosted/ACL/a_basic_operations/acl_level1group_test.py [hunk 1]
```diff
@@ -308,44 +308,4 @@ def test003_level1_and_VMS(self):
             self.lg('-expected error raised %s' % e.message)
             self.assertEqual(e.message,'403 Forbidden')
 
-        """
-    def test004_level1_and_usermangment(self):
```
Comment (x7r3mx) [inline functional_testing/Openvcloud/ovc_master_hosted/ACL/a_basic_operations/acl_level1group_test.py]: why did u remove it?

File: functional_testing/Openvcloud/ovc_master_hosted/ACL/a_basic_operations/acl_usergroup_test.py [hunk 2]
```diff
@@ -170,15 +170,11 @@ def test003_usergroup_delete_user_from_account(self):
         #. try to delete user1 from this account  by himself should be bad request as user is last admin
         #. add user2 to this account by user1 should succeed
         #. delete user1 from this account by user1 should succeed
-        #. delete created account by user1 should be forbidden
```
Comment (x7r3mx) [inline functional_testing/Openvcloud/ovc_master_hosted/ACL/a_basic_operations/acl_usergroup_test.py]: why did u remove it?

File: functional_testing/Openvcloud/ovc_master_hosted/OVC/a_basic/users_tests.py [hunk 3]
```diff
@@ -23,10 +23,6 @@ def test001_authenticate_user(self):
         #. Create user (U1) with admin access.
         #. Authenticate U1,should return session key[user1_key] .
         #. Use U1's key to list the accounts for U1, should succeed.
-        #. Use U1's key to update U1's password, should succeed.
```
Comment (x7r3mx) [inline functional_testing/Openvcloud/ovc_master_hosted/OVC/a_basic/users_tests.py]: why did u remove it?

File: functional_testing/Openvcloud/ovc_master_hosted/OVC/a_basic/users_tests.py [hunk 4]
```diff
@@ -135,43 +118,6 @@ def test003_check_matching_users(self):
         self.assertFalse([x for x in matching_users_names if x["username"]==user2_name ])
         self.assertFalse([x for x in matching_users_names if x["username"]==user1_name ])
 
-    @unittest.skip('https://github.com/0-complexity/openvcloud/issues/955')
-    def test004_password_reset(self):
```
Comment (x7r3mx) [inline functional_testing/Openvcloud/ovc_master_hosted/OVC/a_basic/users_tests.py]: why did u remove it?

Comment (ahmedelsayed-93) [reply functional_testing/Openvcloud/ovc_master_hosted/OVC/a_basic/users_tests.py]: reset user password api is removed

File: functional_testing/Openvcloud/ovc_master_hosted/OVC/a_basic/users_tests.py (hunk 3 above)
Comment (ahmedelsayed-93) [reply functional_testing/Openvcloud/ovc_master_hosted/OVC/a_basic/users_tests.py]: uopdate user password api is removed

File: functional_testing/Openvcloud/ovc_master_hosted/ACL/a_basic_operations/acl_level1group_test.py (hunk 1 above)
Comment (ahmedelsayed-93) [reply functional_testing/Openvcloud/ovc_master_hosted/ACL/a_basic_operations/acl_level1group_test.py]: update && reset user password api is removed

Comment (ahmedelsayed-93) [reply functional_testing/Openvcloud/ovc_master_hosted/ACL/a_basic_operations/acl_level1group_test.py]: /cloudapi/accounts/delete api is removed

File: functional_testing/Openvcloud/ovc_master_hosted/ACL/a_basic_operations/acl_usergroup_test.py (hunk 2 above)
Comment (ahmedelsayed-93) [reply functional_testing/Openvcloud/ovc_master_hosted/ACL/a_basic_operations/acl_usergroup_test.py]: /cloudapi/accounts/delete api is removed

**Pull request merged**